run_marqo.sh

```diff
@@ -1,28 +1,44 @@
 #!/bin/bash
-#source /opt/bash-utils/logger.sh
+
+# set the default value to info and convert to lower case
+export MARQO_LOG_LEVEL=${MARQO_LOG_LEVEL:-info}
+MARQO_LOG_LEVEL=`echo "$MARQO_LOG_LEVEL" | tr '[:upper:]' '[:lower:]'`
+
 export PYTHONPATH="${PYTHONPATH}:/app/src/"
 export CUDA_HOME=/usr/local/cuda/
 export LD_LIBRARY_PATH=${CUDA_HOME}/lib64
 export PATH=${CUDA_HOME}/bin:${PATH}
 
 trap "bash /app/scripts/shutdown.sh; exit" SIGTERM SIGINT
-echo "Python packages:"
-pip freeze
+
+if [ "$MARQO_LOG_LEVEL" = "debug" ]; then
```

Comment:
If we ever decide to make a new level between debug and info, or below debug, it could be worth doing something like `"$MARQO_LOG_LEVEL" <= "debug"`.

Reply:
Currently debug is the lowest debug level that we support: https://docs.marqo.ai/1.3.0/Advanced-Usage/configuration/#configuring-log-level

We would have to implement that mapping logic ourselves. We can implement this if we are required to support a lower level